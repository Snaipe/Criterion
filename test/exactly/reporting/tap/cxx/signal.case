[act]

% signal.cc.bin --tap

[assert]

stdout
  -transformed-by replace_version
  equals -contents-of expected/signal.txt
