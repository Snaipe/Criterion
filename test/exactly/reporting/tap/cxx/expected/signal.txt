TAP version 13
1..3
# Criterion <VERSION>
#
# Running 3 tests from simple
not ok - simple::wrong_signal 
  ---
  assertions: 0
  failures:
  ...
not ok - simple::uncaught unexpected signal after signal.cc:14
ok - simple::caught 
